analytics/management/commands/create_ga_report.py

```diff
@@ -0,0 +1,325 @@
+                            raise KeyError("'{}' not in the list of expected keys for a StreamField".format(key))
+            elif isinstance(body_block.value, list):
+                text = '\n'.join([x.source for x in body_block.value])
+            else:
+                try:
+                    text = body_block.value.source
+                except AttributeError:
+                    text = body_block.value
+            texts.append(text)
+        return texts
+
+    def _get_words(self, text):
+        # Strip out MS-Word specific tags and other junk...probably don't have all of it, but this should handle most...
+        pattern = '<o:DocumentProperties>.*</o:DocumentProperties>|\{mso.*;\}'
+        text, replacements = re.subn(pattern, '', text, flags=re.DOTALL)
+        # Strip HTML tags
+        pattern = '<.*?>'
+        text, replacements = re.subn(pattern, '', text, flags=re.DOTALL)
+        words = re.findall(ur'\w+', text.replace(u'\xa0', ' ').encode('utf-8'))
+        return words
+
+    def _get_statistics_from_database(self, start_date, end_date):
+        data_table = {}
+        articles_of_interest = ArticlePage.objects.filter(first_published_at__range=['2015-08-01', '2016-08-01'])
+        end_date_as_datetime = datetime.strptime(end_date, "%Y-%m-%d")
+        for article in articles_of_interest:
+            data_row = {}
+            data_row['title'] = article.title
+            data_row['url'] = article.url
+            data_row['published_date'] = article.first_published_at
+            data_row['days_published'] = self._calculate_days_published(data_row['published_date'], end_date_as_datetime)
+            data_row['parent_page'] = article.get_parent().title
+            data_row['category'] = article.category.name
+            data_row['is_video'] = article.video
+            data_row['is_interview'] = article.interview
+            data_row['is_visualization'] = article.visualization
+            data_row['word_count_by_type'] = self._calculate_word_count_by_type(article)
+            data_table[data_row['url']] = data_row
+
+        series_of_interests = SeriesPage.objects.filter(first_published_at__range=['2015-08-01', '2016-08-01'])
+        for series in series_of_interests:
+            data_row = {}
+            data_row['title'] = series.title
+            data_row['url'] = series.url
+            data_row['published_date'] = series.first_published_at
+            data_row['days_published'] = self._calculate_days_published(data_row['published_date'], end_date_as_datetime)
+            data_row['parent_page'] = series.get_parent().title
+            data_row['category'] = ''
+            data_row['is_video'] = False
+            data_row['is_interview'] = False
+            data_row['is_visualization'] = False
+            data_row['word_count_by_type'] = self._calculate_word_count_by_type(series)
+            data_table[data_row['url']] = data_row
+
```

Comment:
If by mangle you mean replace, then yes it would. I expect the url to be unique, but I should probably have an assert statement in there just in case.